jest-common/src/main/java/io/searchbox/cluster/TasksInformation.java

```diff
@@ -0,0 +1,42 @@
+package io.searchbox.cluster;
+
+import io.searchbox.action.GenericResultAbstractAction;
+
+/**
+ * Currently only supports retrieving information for a particular task
+ */
+public class TasksInformation extends GenericResultAbstractAction {
```

Comment:
How about doing `AbstractAction<TasksInformationResult>` for specific fields we're using? Future developers can add other fields they need without breaking backward compatibility.